components/ide/jetbrains/launcher/main.go

```diff
@@ -792,6 +792,15 @@ func updateVMOptions(
 	}
 	vmoptions := deduplicateVMOption(ideaVMOptionsLines, gitpodVMOptions, filterFunc)
 
+	// container relevant options
+	vmoptions = append(vmoptions, "-XX:+UseContainerSupport")
+	cpuCount := os.Getenv("GITPOD_CPU_COUNT")
+	parsedCPUCount, err := strconv.Atoi(cpuCount)
+	// if CPU count is set and is parseable as a positive number
+	if err != nil && parsedCPUCount > 0 && parsedCPUCount <= 16 {
+		vmoptions = append(vmoptions, "-XX:ActiveProcessorCount="+cpuCount)
+	}
+
```

Comment:
@akosyakov do you mean something like this?

Reply:
it should be part of gitpodVMOptions, before deduplicateVMOption

Reply:
done